### PR TITLE
use io_utils.print_msg in datset_utils

### PR DESCRIPTION
**Pull request by pedrobrs**
Use of  io_utils.print_msg allows to send messages to python logging by setting:
https://www.tensorflow.org/api_docs/python/tf/keras/utils/disable_interactive_logging
instead of stdout (which remains default behavior if interactive logging is not disabled).
Tested on tf 2.12.

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @pedrobrs Can you please sign CLA. Thank you!

File: keras/utils/dataset_utils.py
Comment (Frightera) [inline keras/utils/dataset_utils.py:673]: Hi @pedrobrs,

Lint is still failing due to inconsistent use of `tab` here. It should be fine if you indent 4 characters using `space`.

**Comment by pedrobrs** [comment]
Would there be anything else that needs to be addressed regarding this?

**Comment by Frightera** [comment]
@pedrobrs You need to request a re-review from @qlzh727, once it's approved it will be merged.

**Comment by gbaned** [comment]
Hi @pedrobrs Can you please address Lint errors? Thank you!

**Comment by pedrobrs** [comment]
I think I fix the problem.   I'm deeply sorry I'm not used to github and I'm not sure if I can run the lints by myself  locally or remote.

**Comment by Frightera** [comment]
@pedrobrs No worries, you can always run that kind of things using Colab. See [this gist](https://colab.research.google.com/gist/Frightera/8b9473f2a6f8e67a41c0d5aaa7f099eb/pr-18054.ipynb).

**Comment by pedrobrs** [comment]
I think that now is ok?

**Comment by Frightera** [comment]
@pedrobrs I don't have necessary permissions to approve a PR on Keras repo - you need to re-request a review from qlzh727. 

Thanks!

**Comment by pedrobrs** [comment]
ok, it said "awaiting review from @qlzh727 "

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**
